lib/vsc/ldap/utils.py

```diff
@@ -67,13 +67,13 @@ def __init__(self, url=None, password=None, connection_dn=None, validation_metho
 
         self.log = getLogger(self.__class__.__name__)
 
-    def tls_settings(self, url):
+    def tls_settings(self):
```

Comment:
you need to change L129 too

Reply:
no. when looking in vsc-ldap-extensions there is a LUMA part which (wrongly) uses this. I don't know if this is used anywhere. but I think it effectively does NOT use TLS, how hard it even tries.

Reply:
actually, If I read the vsc-ldap codes flying around I'll need to keep and use the `url` var. but some of this is blatantly wrong. I'll try to figure out and fix it all.